db/fixtures/aws_instance_types.yml

```diff
@@ -1,220 +1,206 @@
 ---
 a1.2xlarge:
   :architecture:
-  - :x86_64
-  :cluster_networking: 
+  - :arm64
+  :cluster_networking:
   :current_generation: true
   :current_version: true
-  :description: A1 General purpose 2XL
-  :ebs_only: true
```

Comment:
Seems `ebs_only` key went away...will that affect anything?

Reply:
https://github.com/ManageIQ/manageiq-providers-amazon/pull/707#discussion_r641597626
Hence WIP 😄